### DISCUSSION TITLE
Disabled tests appear as unknown in the allure report

### DISCUSSION THREAD

**Discussion by georgecosmin98** [allure-framework/allure2#2034] [General Discussion]
Hello

I disabled my tests with (enabled = false) and in the allure report, disabled tests appear "Unknown". I saw that there would be a possibility to configure the display of the disabled tests, from allure 2.20.0 (https://github.com/allure-framework/allure-java/pull/825). But I couldn't find a concrete example on the internet.

Can you help me with an example?

I tried with:
allure-testng 2.20.0 & allure-testng 2.22.0
testng 7.4.0

Thank you!

**Comment by baev** [comment]
you need to add `allure.testng.hide.disabled.tests=true` to `allure.properties` file

**Reply by georgecosmin98** [reply]
it works, thanks!
